getting-started/README.adoc

```diff
@@ -170,6 +171,60 @@ ifndef::qs[]
 . Verify that your service account was successfully created in the *Streams for Apache Kafka* > *Service Accounts* page.
 endif::[]
 
+[id="proc-setting-service-account-permissions_{context}"]
+== Setting permissions for a service account in a Kafka instance in {product}
+
+{product} uses Access Control Lists (ACLs) provided by Kafka that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create. When you create a service account to connect to a Kafka instance, you must also set the appropriate level of permissions for that new account in the ACL of the instance.
+
+The permissions described in this procedure enable applications associated with the service account to fully interact with the resources in the instance.
```

Comment:
Would this work in a bit more informal style? It feels kind of stuffy for a quick start in the UI. :D 

Let's give your new service account the permissions it needs to allow associated applications to fully interact with the Kafka instance and resources 

(Okay, maybe that's a touch too informal, but ... )

Reply:
Well, as you pointed out in your comment later, I mention it here so that the user knows exactly what the permissions in this example will enable the service account to do. But on second thought, I'd actually rather just delete this sentence and leave what I have later. Users can use this for whatever level of access is appropriate, and the example I've given later is just that. Done.